tests/pytorch_pfn_extras_tests/onnx_tests/utils.py

```diff
@@ -84,6 +86,9 @@ def run_model_test(
             assert len(te_model.graph.output) == len(pfto_model.graph.output)
             assert len(te_model.graph.input) == len(pfto_model.graph.input)
 
+        if check_reconstruct:
+            reconstruct(pfto_model)
+
```

Comment:
I think it would be helpful to have a comparison with the return value of pfto_export in addition to actual when there are no examples of the usage of reconstruct and the validity of the transformation has not been confirmed.